std_msgs/Header header
uint32[2] temps
uint32[7] cell_voltages
uint32 charge_state
uint32[8] fault_bits